### ISSUE TITLE
arangoimport tsv conversion of bools and null, although switched off by `--convert false`

### ISSUE THREAD

**Issue by kpoeppel** [arangodb/arangodb#13169]
## My Environment

* __ArangoDB Version__:       3.7.2 
* __Storage Engine__:             RocksDB
* __Deployment Mode__:       SingleServer
* __Deployment Strategy__:   Manual Start
* __Configuration__:               
* __Infrastructure__:               
* __Operating System__:        Linux Mint
* __Total RAM in your machine__:        16GB
* __Disks in use__:        SDD
* __Used Package__:              

## Component, Query & Data

__Affected feature__:

arangoimport

__AQL query (if applicable)__:
```


```
__AQL explain and/or profile (if applicable)__:

__Dataset__:

__Size of your Dataset on disk__:

__Replication Factor & Number of Shards (Cluster only)__:

## Steps to reproduce

1. echo $'A, B, C, D, E, F, G, H, I\n1, 2, 3, true, false, TRUE, FALSE, null, NULL' > test.csv
2. arangoimport --server.username root --server.password adbpw --create-collection --collection test --type tsv --convert false --server.database smalltests --file test.tsv
3. AQL
```
for t in test
 return {D:is_bool(t.D),E:is_bool(t.E), F:is_bool(t.F),G:is_bool(t.G), H: is_null(t.H), I: is_null(t.I)}
```

__Problem__:
TSV import does type conversion although switched off by '--convert false'

__Expected result__:
"true", "false" and "null" should not be converted and imported as strings.

**Comment by Simran-B** [comment]
The option doesn't seem to be behave as expected, although we are ensure how it was intended to work. But there are definitely inconsistencies.

If you import the following (with double quotes):

```
"A","B","C"
"true","false","null"
```

then the values seem to end up as strings, regardless of the `--convert` option. It's unclear whether this option is supposed to convert quoted values only, or if it's supposed to be completely independent (and quoting reserved solely for "escaping" fields in case they contain the field separator `,`).

**Comment by Simran-B** [comment]
After some more testing, the current behavior is as follows:

| Input | `--convert false` | `--convert true` |
|:--|:--|:--|
| `null` | _(omitted)_ ❗ | _(omitted)_ |
| `"null"` | `"null"` | `"null"` |
| `false` | `false` ❗ | `false` |
| `"false"` | `"false"` | `"false"` |
| `true` | `true` ❗ | `true` |
| `"true"` | `"true"` | `"true"` |
| `12.3` | `"12.3"` | `12.3` |
| `"12.3"` | `"12.3"` | `"12.3"` |

Unquoted literal nulls and booleans are converted regardless of `--convert`, whereas unquoted numbers are either imported as strings or numbers depending on `--convert`. Quoted values are always imported as strings.

I had a discussion with engineering, and while the original intention is still a bit clear, it seems logical to adjust the behavior as follows:

| Input | `--convert false` | `--convert true` |
|:--|:--|:--|
| `null` | `"null"` ✔️ | _(omitted)_ |
| `"null"` | `"null"` | `"null"` |
| `false` | `"false"` ✔️ | `false` |
| `"false"` | `"false"` | `"false"` |
| `true` | `"true"` ✔️ | `true` |
| `"true"` | `"true"` | `"true"` |
| `12.3` | `"12.3"` | `12.3` |
| `"12.3"` | `"12.3"` | `"12.3"` |

Unquoted literal nulls and booleans should be imported as strings if `--convert` is disabled.

Upcoming PR: https://github.com/arangodb/arangodb/pull/13924

**Comment by Simran-B** [comment]
Fixed in 3.6.14, 3.7.11, 3.8.0
